### DISCUSSION TITLE
Appreciation thread

### DISCUSSION THREAD

**Discussion by shinanqu** [oleeskild/obsidian-digital-garden#227] [General]
Hi!

Just want to thank you, Ole Eskild Steensen, so much for creating, maintaining, updating and supporting this remarkable add-on! I really love the Digital Garden! 

I used to have my homepage hosted on WordPress. The first step was creating content in Obsidian and then moving it to WordPress. But it was always a huge huge headache, which lead to a lot of unpublished content. It was just too painful to start WordPress and handle the WordPress editor. It always feels so sluggish, like running a computer from 20 years ago.

Thanks to the Digital Garden, there is no friction anymore. I can simply publish the content instantly from Obsidian with a shortcut. I am still flabbergasted how simple, quick, and easy it is!

I hope you will always have the passion, energy and time to keep the add-on up-to-date and help many content creators to publish their content without any friction!

And thank you, Utsob Roy for helping Ole so much :-) And thank you too everybody else who contributed to the Digital Garden!

Erik

**Comment by oleeskild** [comment]
Thank you for taking the time to write this 😄 
This project was originally just a thing that I wanted for myself. I didn't find any open source solution for publishing only some of my notes in my vault, without having to deal with stuff outside of obsidian. I submitted it to the Obsidian plugin marketplace on a whim in case anyone else found it useful. I'm really glad I did, as the community has been really helpful with suggestions of what to add, and finding obscure bugs that I would never have found myself. It also give me great joy to see that it is used by so many people, and getting these kinds of messages keeps me motivated to continue enhancing it. So thank you, to you, and all the others who have contributed in terms of creating issues, helping others that are stuck and creating PRs. It's been really fun to watch the engagement around it 🎉

**Comment by uroybd** [comment]
This means a lot to me. 😄

The moment I saw this plugin I loved it. Then I thought, why not add features I enjoy for everybody to use? The journey is smooth because @oleeskild is a very nice maintainer (I've seen some really bad one in my life), and our ideas regarding UX aligns very well.

**Comment by wicket-quest** [comment]
@oleeskild : I'm also grateful for your work. I started using Obsidian a few weeks ago. A few days ago, I learned about digital gardens. Your Obsidian plugin lets me start with my digital garden almost immediately. I was stuck, and your plugin helped to solve this.

**Comment by chaosraine** [comment]
Add me to the list. I just started looking into how to get my digital garden online a couple weeks ago and started playing around with a few suggested ideas, but they mostly made it pretty difficult to hide my private notes (the vast majority for now) or required me to basically keep a second copy of a subset of my obsidian files in github which I have to keep organized and updated (which the plugin does, except automatically)
Then I went looking into how to generate an RSS feed and decided it would be too challenging... Until I learned the plugin does that for me too! Plus because it works through a plugin, it's much simpler to actually implement than most of the other options.

Seriously A+ work y'all. I can't thank you enough. If I was going to imagine exactly what I needed from a digital garden publishing platform for Obsidian, it wouldn't be as good as the dg plugin
